cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -34,7 +34,7 @@
 /**
  * Init command for creating a ballerina project.
  */
-@CommandLine.Command(name = INIT_COMMAND, description = "Create a Init Ballerina project")
+@CommandLine.Command(name = INIT_COMMAND, description = "Creates an Init Ballerina project.")
```

Comment:
Shouldn't this be 
```suggestion
@CommandLine.Command(name = INIT_COMMAND, description = "Initializes a Ballerina project.")
```

@hemikak

Reply:
We also seem to have used both initiali**z**e and initiali**s**e. Shall we decide on one?

Reply:
Changed to "Initializes Ballerina project." It should be "initialize" based on the below Google search result.

> Theoretically, there should be separate spellings depending on your system's locale settings. If it's set to Australia, you should get "initialise", while you should get "initialize" if your system is set to the United States.